src/controller/postRouter.js

```diff
@@ -19,6 +19,30 @@ postRouter.post("/", authenticate, validateTextBody, validateScore, async (req,
     }
 });
 
+postRouter.get("/all", authenticate, async (req, res) => {
```

Comment:
Don't need the /all. 

Gets on "/" in a restfulAPI by default should return all the resource data on that path. Ex: get("/posts") would get all posts. If you wanted you could do get("/users/:username") or get("/users/:id") to get all the posts for a specific user depending on whether the post keeps track of its user through the username or id.

I don't think we need auth here, unless we really only want logged in people to see posts